src/general/OptionsMenu.cs

```diff
@@ -600,6 +611,18 @@ private void SwitchMode(OptionsMode mode)
         }
     }
 
+    /// <summary>
+    /// Displays the current viewport resolution
+    /// </summary>
+    private void DisplayResolution()
+    {
+        if (resolution == null)
+            return;
+        var screenResolution = OS.GetScreenSize();
```

Comment:
This is also incorrect. This shows the resolution of the *monitor* the game is on and not the size of the game window / the resolution the game is rendering at.